google/ggrc-core
# GGRC-7161 Enable sort by Notes, Description, Assessment procedure for all objects

# Issue description

Cannot sort by Notes, Description, Assessment procedure in tree view

# Steps to test the changes

Steps to reproduce:
1. Log in ggrc app and open My Work page
2. Open any object tab (Controls)
3. Set Notes, Description, Assessment procedure columns in tree view
4. Sort by values
Actual Result: Sorting is not applied
Expected Result: Sorting should be applied

# Solution description

Modify `tree_view_options` in the js files of the respective objects to enable sorting

# Sanity checklist




# PR Review checklist
